.travis.yml

```diff
@@ -47,6 +47,21 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
+        - pip install --upgrade -r dev-requirements.txt
```

Comment:
I think we can get rid of this line.

Reply:
I tried it and it fails the test. This is the case because `pytest`, `flake8`, and other testing related packages are installed from `dev-requirements.txt`